examples/github.js

```diff
@@ -2,22 +2,18 @@ import apisauce from '../lib/apisauce'
 import R from 'ramda'
 import RS from 'ramdasauce'
 
-const REPO = 'skellock/apisauce'
+const REPO = 'infinitered/apisauce'
 
 const api = apisauce.create({
   baseURL: 'https://api.github.com',
   headers: {
-    Accept: 'application/vnd.github.v3+json'
-  }
+    Accept: 'application/vnd.github.v3+json',
+  },
 })
 
 // attach a monitor that fires with each request
 api.addMonitor(
-  R.pipe(
-    RS.dotPath('headers.x-ratelimit-remaining'),
-    R.concat('Calls remaining this hour: '),
-    console.log
-  )
+  R.pipe(RS.dotPath('headers.x-ratelimit-remaining'), R.concat('Calls remaining this hour: '), console.log),
```

Comment:
I know this is automatic reformatting, but the Elixirist in me thinks that each pipe on its own line looks better!

Reply:
I agree! I might force it to do things differently here.

Reply:
I actually want to de-ramdaify this file as well, as I think it does more harm than good here.

Reply:
Probably not worth forcing the formatting - I was just nitpicking about how sometimes the automated tooling goofs